src/commands/platforms/create.ts

```diff
@@ -99,37 +129,34 @@ export default class PlatformCreate extends BaseCommand {
 
     const account = await AccountUtils.getAccount(this.app, flags.account, { account_message: 'Select an account to register the platform with' });
 
-    const platform = await this.createArchitectPlatform(flags);
+    const contexts = await this.setupKubeContext(flags);
 
-    const platform_dto = { name: platform_name, ...platform, flags: flags_map };
+    try {
+      const platform = await this.createArchitectPlatform(flags, contexts.current_context);
```

Comment:
As this isn't a `Platform` and is `CreatePlatformInput`, naming this `create_platform_input` would be more helpful